history/characters/mechagnome.txt

```diff
@@ -0,0 +1,138 @@
+﻿attendant_tock={
+	name=Tock
+	#dynasty=10110
+	culture=mechagnome religion=mystery_of_the_makers
+	martial=1 diplomacy=3 stewardship=7 intrigue=1 learning=7
+	trait=education_stewardship_4 trait=intellect_good_3 trait=content trait=diligent trait=calm trait=patient trait=loyal
+	disallow_random_traits = yes
+	2.11.12={ birth=yes trait=creature_mechagnome }
+	50.4.2={ 
+		employer=400505
+		effect = { 
+			add_opinion = { modifier = loyal_servant target = character:400505 }
+		}
+		learn_language = language_gnomish
+	}
+	1700.3.30={ death=yes }
+}
+archivist_mechaton={
+	name=Mechaton
+	culture=mechagnome religion=mystery_of_the_makers
+	martial=7 diplomacy=1 stewardship=7 intrigue=1 learning=7
+	trait=education_learning_4 trait=intellect_good_3 trait=content trait=diligent trait=calm trait=patient trait=loyal
+	disallow_random_traits = yes
+	2.11.12={ birth=yes trait=creature_mechagnome }
+	50.4.2={ 
+		employer=400505
+		effect = { 
+			add_opinion = { modifier = loyal_servant target = character:400505 }
+		}
+		learn_language = language_gnomish
+	}
+	1700.3.30={ death=yes }
+}
+dirkee={
+	name=Dirkee
+	culture=mechagnome religion=mystery_of_the_makers
+	martial=7 diplomacy=1 stewardship=6 intrigue=1 learning=5
+	trait=education_martial_2 trait=intellect_good_3 trait=lunatic_1 trait=disloyal
+	trait=wrathful trait=diligent trait=arbitrary
+	disallow_random_traits = yes
+	2.11.12={ birth=yes trait=creature_mechagnome }
```

Comment:
employer?

Reply:
I don't think he has any

Reply:
so he is just gonna wander the map? what happens when you dont give someone an employer?

Reply:
They either wander around or become courtiers to a ruler that is closest to their religion and culture